calculator/src/main/java/com/example/aperez/onboarding/mvp/model/CalculatorModel.java

```diff
@@ -0,0 +1,30 @@
+package com.example.aperez.onboarding.mvp.model;
+
+/**
+ * Created by aperez on 23/01/17.
+ */
+
+public class CalculatorModel {
+
+    private Double result;
```

Comment:
Maybe, you could return individual values in each operation, yo avoid clean the result value in each new operations. I think.